### PR TITLE
Refactor leftover metric keys

### PR DESCRIPTION
**Pull request by LuQQiu**
Refactor async cache and block related metric keys

### CODE REVIEW

**Comment by LuQQiu** [comment]
Tests in error: 
alluxio.client.cli.fs.command.CopyFromLocalCommandIntegrationTest.copyFromLocalLarge(alluxio.client.cli.fs.command.CopyFromLocalCommandIntegrationTest)
  Run 1: CopyFromLocalCommandIntegrationTest.copyFromLocalLarge:175 » Internal http2 ex...
  Run 2: CopyFromLocalCommandIntegrationTest.copyFromLocalLarge:175 » Internal http2 ex...

**Comment by LuQQiu** [comment]
Jenkins, test this please

**Comment by LuQQiu** [comment]
@madanadit PTAL, thanks

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 1]
```diff
@@ -533,12 +533,85 @@ public MetricKey build() {
           .build();
 
   // Worker metrics
+  public static final MetricKey WORKER_ASYNC_CACHE_DUPLICATE_REQUESTS =
+      new Builder(Name.WORKER_ASYNC_CACHE_DUPLICATE_REQUESTS)
+          .setDescription("Total number of duplicated async cache request received by this worker")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggreagated(false)
```
[the remark this answers is not in the document]
Comment (LuQQiu) [reply core/common/src/main/java/alluxio/metrics/MetricKey.java]: Oh!!! Thanks for finding it!
This is used when worker/client report metrics. They only report the `isClusterAggregated == true` metrics. This is helpful to reduce the metrics client sent and leading master stored.

Now I only define those metrics that are actually aggregated to cluster metrics in `DefaultMetricsMaster` with `isClusterAggregated == true`. In the future, maybe all the metrics that we think can be aggregated at cluster level can set it to true.

**Comment by LuQQiu** [comment]
Tests in error: 
alluxio.server.ft.journal.MultiMasterJournalTest.testCheckpointReplay(alluxio.server.ft.journal.MultiMasterJournalTest)
  Run 1: MultiMasterJournalTest.testCheckpointReplay:64 » IO java.util.concurrent.Timeo...
  Run 2: MultiMasterJournalTest.before:53 » IO java.util.concurrent.TimeoutException: T...

**Comment by LuQQiu** [comment]
Jenkins, test this please

**Comment by madanadit** [comment]
```
Tests in error: 
alluxio.server.ft.journal.MultiMasterJournalTest.testCheckpointReplay(alluxio.server.ft.journal.MultiMasterJournalTest)
  Run 1: MultiMasterJournalTest.testCheckpointReplay:64 » IO java.util.concurrent.Timeo...
  Run 2: MultiMasterJournalTest.before:53 » IO java.util.concurrent.TimeoutException: T...
```

**Comment by madanadit** [comment]
Jenkins, test this please

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**